docs/user/admin/settings.rst

```diff
@@ -384,3 +384,129 @@ Result set::
 	  "timed_out" : false
 	}
 
+opendistro.sql.cursor.enabled
+=============================
+
+Description
+-----------
+
+User can enable/disable pagination for all queries that are supported.
+
+1. The default value is false.
+2. This setting is node scope.
```

Comment:
what is the meaning of "node scope"? does it means the setting only apply to specific node?

Reply:
> what is the meaning of "node scope"? 

This is a cluster wide setting, change on one node will be reflected on all other nodes.
Not sure, why it is named like this. This is not different from all other settings we already have.

> does it means the setting only apply to specific node?

No.

Reply:
I may have a little context here. I recall the name was from ES `Setting` in which there is a NodeScope and IndexScope.